example/collector-k8-manifest.yml

```diff
@@ -19,13 +19,19 @@ spec:
         verbosity: detailed
       loki:
         endpoint: "http://loki.default.svc:3100/loki/api/v1/push"
-
+    processors:
+      filter/ottl:
+        error_mode: ignore
+        logs:
+          log_record:
+          - 'not (IsMatch(body, ".*Permission denied.*"))'
```

Comment:
@Shashankft9 Thank you for the PR! :rocket: 
Though this method to filter logs will work however it shouldn't be handled this way ideally. It's because there is logic for filtering already present in Kubearmor.

Kubearmor emits separate streams for logs (generic events) and alerts (policy events). So a good place to investigate will be why the adapter can't subscribe to just a single stream based on the configuration passed.
The relevant code - https://github.com/kubearmor/otel-adapter/blob/9f702ef1f52dd6e161061908b2bc884f281bb473/stanza_input_operator/input_operator.go#L131